src/Files.Uwp/Themes/Nightshade.xaml

```diff
@@ -0,0 +1,50 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:BelowWindows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)">
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <!--  Background Resources  -->
+            <SolidColorBrush x:Key="RootBackgroundBrush" Color="#2c303b" />
+            <Color x:Key="SolidBackgroundAcrylic">#535b70</Color>
```

Comment:
Can you move the brush to be right after `SolidBackgroundFillColorQuarternaryBrush`? This will help keep the themes consistent with each other.

Reply:
OK.